content/en/account_management/billing/rum.md

```diff
@@ -30,3 +30,7 @@ Browser logs are automatically tied to the corresponding RUM session, enabling y
 ## Does Datadog use cookies?
 
 Yes. Datadog uses cookies to stitch together the various steps of your users into a session. This process does not use cross-domain cookies, and it does not track the actions of your users outside your applications.
+
+## I am seeing Session Replays showing up in my usage page, but I have not configured Session Replay in my environment.
+
+Session Replay is the name of a SKU. Compared to the Browser SKU, the Session Replay SKU also includes replays, long tasks, and resources. So, if you are collecting any of the three, you will be billed for the Session Replay SKU. 
```

Comment:
```suggestion
Session Replay includes replays, long tasks, and resources. If you are collecting any of the three, you will be billed for Session Replay.
```

Reply:
@MirandaKapin what do you think?

Reply:
More context to this edit: we do not use the term SKU or contextual billing terminology in the documentation. We should reference as the product name(s).

Reply:
understood, that makes sense. thank you for clarifying